src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -45,39 +46,54 @@ public KeysController(WebScriptHostManager scriptHostManager, ISecretManager sec
         }
 
         [HttpGet]
-        [Route("admin/host/keys")]
+        [Route("admin/host/{keys:regex(^(keys|functionkeys|systemkeys)$)}")]
```

Comment:
this route match will be case insensitive, right? e.g. functionKeys is supported